interfaces/inc/c_pal/threadpool.h

```diff
@@ -21,10 +21,13 @@ extern "C" {
 #endif
 
 typedef struct THREADPOOL_TAG THREADPOOL;
+typedef struct TIMER_INSTANCE_TAG TIMER_INSTANCE;
 typedef struct TIMER_INSTANCE_TAG* TIMER_INSTANCE_HANDLE;
```

Comment:
>TIMER_INSTANCE_HANDLE [](http://example.com/codeflow?start=35&length=21)

Who uses the TIMER_INSTANCE_HANDLE? I am imagining that after this change there should be only one, right?

Reply:
removed the timer_instance_handle, not using anymore